docker/controller-node/jobs/test-pipeline-ruby.cps

```diff
@@ -0,0 +1,22 @@
+pipeline {
+    agent {
+        label 'agent-ruby'
+    }
+    stages {
+        stage('Checkout') {
+            steps {
+                git url: 'git@github.com:freeCodeCamp/devdocs.git',
+                        credentialsId: 'github-ssh'
```

Comment:
```suggestion
                        credentialsId: 'github-ssh', branch: 'main'
```
The checkout fails for me locally unless I specify the branch. The default name is `master`, and that doesn't exist in the checked out repo